.github/workflows/ci.yml

```diff
@@ -9,6 +9,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
```

Comment:
If locking the node version is required, a comment indicating what happened when it wasn't locked could be useful.

Reply:
Thanks for pointing this out, it looks like it might not be necessary. I'm going to release a version without it.